File: app/src/main/java/org/dhis2/usescases/notes/NotesComponent.java
package org.dhis2.usescases.notes;

import org.dhis2.commons.di.dagger.PerFragment;

import dagger.Subcomponent;

@PerFragment
@Subcomponent(modules = NotesModule.class)
public interface NotesComponent {

    void inject(NotesFragment notesFragment);

}

File: app/src/main/java/org/dhis2/usescases/teiDashboard/dashboardfragments/notes/NotesComponent.kt
package org.dhis2.usescases.teiDashboard.dashboardfragments.notes

import dagger.Subcomponent
import org.dhis2.data.dagger.PerFragment

@PerFragment
@Subcomponent(modules = [NotesModule::class])
interface NotesComponent {
    fun inject(notesFragment: NotesFragment)
}
